File: domain/src/main/java/com/icehousecorp/maunorafiq/domain/forecast/repository/ForecastRepository.java
package com.icehousecorp.maunorafiq.domain.forecast.repository;

import com.icehousecorp.maunorafiq.domain.forecast.Forecast;

import rx.Observable;

/**
 * Created by maunorafiq on 11/29/16.
 */

public interface ForecastRepository {
    Observable<Forecast> forecastWeather(String city);
}

File: domain/src/main/java/com/icehousecorp/maunorafiq/domain/forecast/repository/ForecastRepository.kt
package com.icehousecorp.maunorafiq.domain.forecast.repository

import com.icehousecorp.maunorafiq.domain.forecast.Forecast

import rx.Observable

/**
 * Created by maunorafiq on 11/29/16.
 */

interface ForecastRepository {
    fun forecastWeather(city: String): Observable<Forecast>
}
